cogs/utils/db.py

```diff
@@ -172,6 +172,24 @@ async def set_member(guildID, memberID, option, value):
     await conn.close()
 
 
+# Returns members who have birthday on a given date
+async def get_members_by_bday(date):
+    conn = await db_connect()
+    c = await conn.cursor()
+    date_valid_format = f"%{date.strftime('%m')}-{date.strftime('%d')}%"
+    date_invalid_format = f"%{date.month}-{date.day}%"
+    sql = f"""
+    SELECT *
```

Comment:
This should probably still be `SELECT DISTINCT member_id` to avoid pulling members who happen to be on two different servers with the same bot and the same birthday entered for both profiles. My SQL knowledge isn't super great so correct me if I'm wrong on that usage!

Reply:
That's why I added `GROUP BY member_id` at the end to avoid duplicates. I've used `SELECT *` to get every column from the member if any other data will be needed in the future.

Reply:
Oh my mistake, I missed the addition of the `GROUP BY` line. Looks good :+1: